### PR TITLE
Fix #2516. Remove the runner code.

### LINKED ISSUE #2516: Remove runner
It's obsolete.

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

File: sirepo/server.py [hunk 1]
```diff
@@ -631,9 +631,7 @@ def init(uwsgi=None, use_reloader=False):
 def init_apis(app, *args, **kwargs):
     for e, _ in simulation_db.SCHEMA_COMMON['customErrors'].items():
         app.register_error_handler(int(e), _handle_error)
-    importlib.import_module(
-        'sirepo.' + ('job' if feature_config.cfg().job else 'runner')
-    ).init_by_server(app)
+    importlib.import_module('sirepo.job').init_by_server(app)
```
Comment (robnagler) [inline sirepo/server.py:634]: Don't need import_module, just use `import sirepo.job`

File: sirepo/simulation_db.py [hunk 2]
```diff
@@ -297,30 +295,6 @@ def generate_json(data, pretty=False):
     return util.json_dump(data, pretty=pretty)
 
 
-def hack_nfs_write_status(status, run_dir):
```
Comment (robnagler) [inline sirepo/simulation_db.py:300]: Yay!

File: sirepo/simulation_db.py [hunk 3]
```diff
@@ -483,13 +457,6 @@ def prepare_simulation(data, run_dir=None):
         list, py.path: pkcli command, simulation directory
     """
     import sirepo.sim_data
-    if run_dir is None:
```
Comment (robnagler) [inline sirepo/simulation_db.py:486]: is this called anywhere without a run_dir?

File: sirepo/simulation_db.py [hunk 4]
```diff
@@ -263,9 +263,7 @@ def get_schema(sim_type):
     pkcollections.mapping_merge(
```
Comment (robnagler) [inline sirepo/simulation_db.py:263]: Could this just be schema.feature_config = ...

File: sirepo/server.py (hunk 1 above)
Comment (e-carlin) [reply sirepo/server.py]: https://github.com/radiasoft/sirepo/pull/2532/commits/e8d7a943f378eb6fdb6a9315cea4522022b14197

File: sirepo/simulation_db.py (hunk 3 above)
Comment (e-carlin) [reply sirepo/simulation_db.py:486]: No. I've made the argument not optional. https://github.com/radiasoft/sirepo/pull/2532/commits/e8d7a943f378eb6fdb6a9315cea4522022b14197

File: sirepo/simulation_db.py (hunk 4 above)
Comment (e-carlin) [reply sirepo/simulation_db.py:263]: https://github.com/radiasoft/sirepo/pull/2532/commits/e8d7a943f378eb6fdb6a9315cea4522022b14197

**Comment by e-carlin** [comment]
@robnagler all comments addressed.

**Pull request merged**